libs/api-test-environment/src/main/java/de/cyface/apitestutils/ApiServer.java

```diff
@@ -107,12 +106,12 @@ public void start(final Vertx vertx, final VertxTestContext testContext, final T
 
         final var privateTestKey = this.getClass().getResource("/private_key.pem");
         final var publicTestKey = this.getClass().getResource("/public.pem");
-        config.put(Parameter.MONGO_DATA_DB.key(), mongoDatabase.config())
-                .put(Parameter.MONGO_USER_DB.key(), mongoDatabase.config())
-                .put(Parameter.HTTP_PORT.key(), port)
-                .put(Parameter.JWT_PRIVATE_KEY_FILE_PATH.key(), Validate.notNull(privateTestKey).getFile())
-                .put(Parameter.JWT_PUBLIC_KEY_FILE_PATH.key(), Validate.notNull(publicTestKey).getFile())
-                .put(Parameter.HTTP_HOST.key(), HTTP_HOST)
+        config.put("mongo.datadb", mongoDatabase.config())
+                .put("mongo.userdb", mongoDatabase.config())
+                .put("http.port", port)
+                .put("jwt.private", Validate.notNull(privateTestKey).getFile())
+                .put("jwt.public", Validate.notNull(publicTestKey).getFile())
+                .put("http.host", HTTP_HOST)
```

Comment:
This was necessary to make the `libs/api-test-environment` not dependent of `libs/api` in order to move all tests of `libs/api` to `libs/api` (also the ones which depend on `libs/api-test-environment` like the Authorization test we talked about a week or two ago)